### ISSUE TITLE
"netlab libvirt package" does not work for EOS, IOSv, and CSR under Ubuntu 22.04

### ISSUE THREAD

**Issue by ipspace** [ipspace/netlab#485]
FYI, the `netlab libvirt package eos vEOS64-lab-4.28.2F.qcow2` gives an error for deprecated option :

```
We'll start the VM from the newly-created virtual disk. When the
VM starts, execute 'netlab libvirt config eos' in another
window and follow the instructions.

WARNING  --os-type is deprecated and does nothing. Please stop using it.
ERROR    
--os-variant/--osinfo OS name is required, but no value was
set or detected.

This is now a fatal error. Specifying an OS name is required
for modern, performant, and secure virtual machine defaults.

You can see a full list of possible OS name values with:

   virt-install --osinfo list

If your Linux distro is not listed, try one of generic values
such as: linux2020, linux2018, linux2016

If you just need to get the old behavior back, you can use:

  --osinfo detect=on,require=off

```

_Originally posted by @htam-net in https://github.com/ipspace/netlab/discussions/408#discussioncomment-3711634_

**Comment by jbemmel** [comment]
Another solution might be to use cEOS with Containerlab: https://containerlab.dev/manual/kinds/ceos/

**Comment by ipspace** [comment]
And what do you plan to do with Junos, Nexus OS, VyOS, Dell OS10, IOS XE, IOS... ;)

**Comment by jbemmel** [comment]
JunOS: [cRPD](https://containerlab.dev/manual/kinds/crpd/), vMX or vQFX
Nexus: [Nexus9000v](https://containerlab.dev/manual/kinds/vr-n9kv/)
Dell OS10: https://containerlab.dev/manual/kinds/vr-ftosv/
Other Ciscos: https://containerlab.dev/manual/kinds/vr-xrv/, https://containerlab.dev/manual/kinds/vr-xrv9k/, https://containerlab.dev/manual/kinds/vr-csr/

Only VyOS isn't listed there yet, but something like https://github.com/ravens/docker-vyos might work

**Comment by ipspace** [comment]
All those are VMs running inside containers -- "solving" the problem by asking the user to install yet another toolchain (vrnetlab) and applying RFC 1925 rule 6a. 👍

**Comment by ipspace** [comment]
After spending a few hours looking at various installation screens:

* Fresh install of Ubuntu 20.04 LTS still works. Fixed the documentation to point out that "minor detail" (1cb6778b6a16327d77db7f0f477ff5a9281d91ee)
* Looks like Ubuntu 22.04 LTS uses a completely different version of _libvirt_. It doesn't want to use the old CLI parameters, and the suggested new CLI parameters don't work on Ubuntu 20.04 which is our primary target at the moment.

Will eventually figure out what to do (although it looks like IOSv won't work at all with newer KVM/QEMU versions), but this is definitely not a critical problem (yet) for users following the installation instructions.

**Comment by ipspace** [comment]
Fixed in 9fc2ebe15753f3cd83168d7f80092edfa4ca2575
